misc/python/materialize/cli/scratch/login.py

```diff
@@ -0,0 +1,21 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+import argparse
+
+from materialize import spawn
+from materialize.cli.scratch import check_required_vars
+
+
+def configure_parser(parser: argparse.ArgumentParser) -> None:
+    check_required_vars()
+
+
+def run(args: argparse.Namespace) -> None:
+    spawn.runv(["aws", "sso", "login"])
```

Comment:
is this missing the correct profile?

Reply:
Nope it auto picks it up from the AWS_PROFILE var that gets set in check_required_vars!